packages/dd-trace/src/plugins/server.js

```diff
@@ -4,6 +4,8 @@ const InboundPlugin = require('./inbound')
 
 class ServerPlugin extends InboundPlugin {
   static get operation () { return 'request' }
+  static get kind () { return 'server' }
+  static get type () { return 'web' } // a default that may eventually be overriden by nonweb servers
```

Comment:
This should be a different class. A server is not always a web server (as pointed out by the comment).